lib/models.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'json'
+require 'net/http'
+require 'rest-client'
```

Comment:
Not recommend to use this gem, it is outdated
https://github.com/rest-client/rest-client